HelpSource/Classes/UnitTest.schelp

```diff
@@ -107,63 +107,92 @@ YourClass.test
 ::
 Runs the test class for code::YourClass::, which is assumed to be named code::TestYourClass::.
 
-If no test class if found it politely declines.
+If no test class is found, it politely declines.
 
 note::
-UnitTests for the Common library classes are kept in the CommonTests quark. 	This enables you to easily install and uninstall these tests.
+UnitTests for the Common library classes are kept in the CommonTests quark. This enables you to easily install and uninstall these tests.
```

Comment:
This is no longer true. `CommonTests` is deprecated and the test suite now exists in the main repository under `testsuite`. We have plans to eventually include it in the standard distro.